docroot/modules/humsci/hs_bugherd/src/ConfigOverrides.php

```diff
@@ -0,0 +1,97 @@
+<?php
```

Comment:
Shouldn't this file be in `src/Overrides/Configoverrides.php`

Reply:
I thought so too. but this guide doesnt https://www.drupal.org/docs/8/api/configuration-api/configuration-override-system#providing-overrides-from-modules

Reply:
The documentation says just in `src` directory. 🤷‍♂️ 
 https://www.drupal.org/docs/8/api/configuration-api/configuration-override-system#providing-overrides-from-modules

Reply:
I don't see anywhere it explicitly says it needs to be the first level of the `src` directory but rather, the example doesn't put it in any nested folder. Would it make more sense to have it placed in a directory in case more overrides are created in the future?

Reply:
no argument here 👍